utils/json.go

```diff
@@ -94,6 +95,9 @@ func GetUpdatedJson(old interface{}, new interface{}, option UpdateJsonOption) i
 			if option.IgnoreCasing && strings.EqualFold(oldValue, newStr) {
 				return oldValue
 			}
+			if option.IgnoreMissingProperty && regexp.MustCompile(`^\**$`).MatchString(newStr) {
```

Comment:
What if there is an attribute that is set as `***` as input? Also, should the regexp be `^\*+$`, as otherwise empty string will also be matched?

Reply:
> What if there is an attribute that is set as *** as input?
This should not be a common case, user sets "***" but it returns different amount of *.

> should the regexp be ^\*+$, as otherwise empty string will also be matched?
Yes, I've updated it.